libs/wire-api-federation/src/Wire/API/Federation/API/Galley.hs

```diff
@@ -74,8 +74,18 @@ data ConversationMemberUpdate = ConversationMemberUpdate
   { cmuTime :: UTCTime,
     cmuOrigUserId :: Qualified UserId,
     cmuConvId :: Qualified ConvId,
-    cmuAlreadyPresentUsers :: [UserId], -- pre-existing users in the conversation from the receiving domain
+    -- | This field notifies a remote backend about which users need to be sent
```

Comment:
A field doesn't "notify". Maybe just write something like "users from this backend who need to ..."

Reply:
Thanks! Incorporating that...